client.go

```diff
@@ -0,0 +1,168 @@
+// Package kafkatest provides a package intended for running tests
+// that require a Kafka backend.
+package kafkatest
+
+import (
+	"crypto/rand"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/Shopify/sarama"
+)
+
+var ErrDisabled = fmt.Errorf("kafka tests are disabled")
+
+// New connects to a Kafka instance and returns a Kafka
+// instance that uses it.
+//
+// The following environment variables can be used to
+// configure the connection parameters:
+//
+//	- $KAFKA_DISABLE
+//		A boolean as parsed by strconv.ParseBool. If this is true,
+//		New will return ErrDisabled.
+//	- $KAFKA_ADDRS
+//		A comma-separate list of Kafka broker addresses in host:port
+//		form. If this is empty, localhost:9092 will be used.
+//		The list of address can be discovered by calling Client.Addrs.
+//	- $KAFKA_USE_SASL
+//		A boolean as parsed by strconv.ParseBool. If this is true,
+//		a secure SASL connection will be used to the Kafka brokers.
+//	- $KAFKA_SASL_USERNAME, $KAFKA_SASL_PASWORD
+//		The SASL username and password to use. Ignored if
+//		$KAFKA_USE_SASL isn't true.
+//	- $KAFKA_USE_TLS
+//		A boolean as parsed by strconv.ParseBool. If this
+//		is true, a secure TLS connection will be used.
+//
+// The returned Kafka instance must be closed after use.
+func New() (*Kafka, error) {
+	disabled, err := boolVar("KAFKA_DISABLE")
+	if err != nil {
+		return nil, fmt.Errorf("bad value for $KAFKA_DISABLE: %v", err)
```

Comment:
🌷 I would say what kind of value is expected there.

Something like `allowed values for $KAFKA_DISABLE are [true|false] but current value is %v`

Reply:
fixed (in the boolVar function).